be/src/exec/pipeline/olap_chunk_source.h

```diff
@@ -136,10 +136,14 @@ class OlapChunkSource final : public ChunkSource {
     RuntimeProfile::Counter* _pred_filter_counter = nullptr;
     RuntimeProfile::Counter* _del_vec_filter_counter = nullptr;
     RuntimeProfile::Counter* _pred_filter_timer = nullptr;
+    RuntimeProfile::Counter* _vec_pred_filter_timer = nullptr;
```

Comment:
Do we really need these new filter metrics？

Reply:
> Do we really need these new filter metrics？

I think these may be helpful for our later optimization. Further, maybe we need to consider classifying runtime profiles in the future? These metrics are too detailed and not necessary all the time

Reply:
I see，but currently all metrics are exposed to users, so we should add the metric only if necessary. these metrics are too detail. 
> maybe we need to consider classifying runtime profiles in the future? 

I agree  with you

Reply:
> I see，but currently all metrics are exposed to users, so we should add the metric only if is necessary. these metrics are too detail.

I have removed them.